tools/generator/page-builder-admin.js

```diff
@@ -0,0 +1,635 @@
+              <span class="stat-chip drifted ${stats.drifted === 0 ? 'zero' : ''}">${stats.drifted} drifted</span>
+              <span class="stat-chip new ${stats.new === 0 ? 'zero' : ''}">${stats.new} new</span>
+              <span class="stat-chip stale ${stats.stale === 0 ? 'zero' : ''}">${stats.stale} stale</span>
+              <span class="stat-chip blocked ${stats.blocked === 0 ? 'zero' : ''}">${stats.blocked} blocked</span>
+            </div>
+          ` : nothing}
+        </div>
+        <div class="toolbar-right">
+          ${hasSheets ? html`
+            <sl-button size="small" variant="primary" ?disabled=${isBlocked} @click=${this.handleSyncSheet}>⚡ Sync Sheet</sl-button>
+            <sl-button size="small" variant="${isBlocked ? 'success' : 'default'}" @click=${this.handleToggleSheetBlock}>${isBlocked ? '🔓 Unblock' : '🔒 Block'}</sl-button>
+          ` : html`<sl-button size="small" variant="primary" @click=${this.handleSyncAll}>⚡ Sync All</sl-button>`}
+        </div>
+      </div>
+    `;
+  }
+
+  renderItemRow(item) {
+    const isSheetBlocked = this.isSheetBlocked(item.sheetName);
+    const displayState = isSheetBlocked ? ITEM_STATES.BLOCKED : item.state;
+    const canSync = [ITEM_STATES.DRIFTED, ITEM_STATES.NEW].includes(item.state);
+
+    return html`
+      <div class="item-row ${displayState}">
+        <div class="item-col item-state"><span class="state-badge ${displayState}">${ITEM_STATE_LABELS[displayState]}</span></div>
+        <div class="item-col item-value">${item.currentValue}${item.live && item.live.value !== item.currentValue ? html`<span class="live-hint">Live: ${item.live.value}</span>` : nothing}</div>
+        <div class="item-col item-label">${item.currentLabel}${item.live && item.live.label !== item.currentLabel ? html`<span class="live-hint">Live: ${item.live.label}</span>` : nothing}</div>
+        <div class="item-col item-actions">
+          <button class="action-btn sync-btn" ?disabled=${!canSync || isSheetBlocked} @click=${() => this.handleSyncItem(item.itemKey)}>Sync</button>
+          <button class="action-btn ${item.blocked ? 'unblock-btn' : 'block-btn'}" ?disabled=${isSheetBlocked} @click=${() => this.handleToggleBlock(item.itemKey)}>${item.blocked ? 'Unblock' : 'Block'}</button>
+          ${item.state === ITEM_STATES.STALE ? html`<button class="action-btn delete-btn" ?disabled=${isSheetBlocked} @click=${() => this.handleRemoveItem(item.itemKey)}>Delete</button>` : nothing}
+        </div>
+      </div>
+    `;
+  }
+
+  renderItemList(items) {
+    const isBlocked = this.isSheetBlocked(this.activeSheet);
+    if (items.length === 0) return html`<div class="empty-state"><div class="empty-icon">📝</div><h3>No items in this sheet</h3></div>`;
+
+    return html`
+      ${isBlocked ? html`<div class="blocked-sheet-banner">🔒 This sheet is locked and will not be synced</div>` : nothing}
```

Comment:
maaay prefer our iconography

Reply:
It's a tool mostly for us but I'm just going to remove the icons, they don't add value.